src/languages/es.js

```diff
@@ -961,4 +961,22 @@ export default {
         },
         refresh: 'Refresh',
     },
+    fileDownload: {
+        success: {
+            title: 'Descargado!',
+            message: 'Adjunto descargado con éxito',
+            buttonOkText: 'OK',
+        },
+        generalError: {
+            title: 'Error de archivo adjunto',
+            message: 'No se puede descargar el archivo adjunto',
+            buttonCancelText: 'Cancelar',
+        },
+        permissionError: {
+            title: 'Acceso necesario',
+            message: 'NewExpensify no tiene acceso para guardar archivos adjuntos. Para habilitar el acceso, toque Configuración y permita el acceso.',
+            buttonCancelText: 'Cancelar',
+            buttonSettingsText: 'Ajustes',
+        },
+    },
```

Comment:
Could you please confirm these on slack?

Reply:
Updated